app/controllers/pages_controller.rb

```diff
@@ -1,12 +1,13 @@
 class PagesController < ApplicationController
-  before_action :fetch_form, :convert_session_keys_to_symbols
```

Comment:
`fetch_form` has been removed but it didn't need to be added to the before action it it was a proper memorised method like `current_form` which creates the instance variable the first time its called rather than always calling it even if it is not used.